src/main/keep-container/keep-content/keep-common-note/keep-common-note.component.html

```diff
@@ -0,0 +1,88 @@
+<div *ngIf="(notes$ | async) as notes" class="notes grey-background" #notes>
+  <div *ngFor="let note of notes$ |async" [@noteEnterLeave]="note">
+
+    <div class="note"
+         *ngIf="(isSearch && note.noteExist && (isArchiveNotePresent ? note.isArchived : !note.isArchived) ) || (!isSearch && (isArchiveNotePresent ? note.isArchived : !note.isArchived))"
```

Comment:
can we have a function here instead of complex condition here

Reply:
updated